app/src/main/res/values/attr.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <declare-styleable name="StarView">
+        <attr name="line_color" format="color" />
```

Comment:
@ifeins in R.styleable it adds what you say automatically (you can see above `R.styleable.StarView_line_color`)